### PR TITLE
Remove Chart/Dim based communication

### PR DESCRIPTION
**Pull request by underhood**
##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: database/sqlite/sqlite_aclk.c [hunk 1]
```diff
@@ -455,52 +406,12 @@ static void timer_cb(uv_timer_t* handle)
...
-    stop_retention_run();
-    wc->retention_running = 0;
-
-    struct aclk_database_cmd cmd;
-    memset(&cmd, 0, sizeof(cmd));
-    cmd.opcode = ACLK_DATABASE_DIM_DELETION;
-    if (aclk_database_enq_cmd_noblock(wc, &cmd))
-        info("Failed to queue a dimension deletion message");
-
-    cmd.opcode = ACLK_DATABASE_NODE_INFO;
```
Comment (MrZammler) [inline database/sqlite/sqlite_aclk.c:473]: Hi Timo!

This was piggy-backed there to do a NODE INFO send every `ACLK_DATABASE_ROTATION_INTERVAL`. Not this PR's problem I think, as long as we do a follow up one (I can do it) to have the functionality again.

Comment (stelfrag) [reply database/sqlite/sqlite_aclk.c:473]: It can go to the CLEANUP interval for now -- A follow up PR will need to clean the tables completely

**Comment by stelfrag** [comment]
Function `sql_maint_aclk_sync_database` needs to handle only `aclk_alert_xxx` tables

File: database/sqlite/sqlite_aclk.c (hunk 1 above)
Comment (stelfrag) [reply database/sqlite/sqlite_aclk.c:473]: I added this

**Review by stelfrag — APPROVED** [review]

**Comment by underhood** [comment]
@MrZammler did you take a peek after the last changes?

**Comment by MrZammler** [comment]
> @MrZammler did you take a peek after the last changes?

Yes, sorry for the delay...

**Review by MrZammler — APPROVED** [review]

**Pull request merged**